lua/lspconfig/configs/kotlin_language_server.lua

```diff
@@ -2,7 +2,7 @@ local util = require 'lspconfig.util'
 
 local bin_name = 'kotlin-language-server'
 if vim.fn.has 'win32' == 1 then
-  bin_name = bin_name .. '.bat'
+  bin_name = bin_name .. '.cmd'
 end
```

Comment:
can't we just delete this entire block? cc @glepnir

Reply:
Yes I think so. Also other configs. Because on my machine it still works when not declaring extension.

![image](https://github.com/user-attachments/assets/b83340b8-f2d6-4e11-aa64-5ce91e8a17ef)

Reply:
I just changed .bat to .cmd because I see other config check for win32 to add extension

Reply:
> I see other config check for win32 to add extension

those are probably just old

Reply:
cmd already process by exepath. no need if block

Reply:
> cmd already process by exepath. no need if block

So I will remove the if block then?

Do you want me to create another PR for removing if block also?

Reply:
> So I will remove the if block then?

yes



> Do you want me to create another PR for removing other configs' if block also?

sure, unless @glepnir knows of a reason we didn't do that already